calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -209,6 +221,7 @@ def _get_api_path(self, path):
         :return: A list of JSON API objects
         :rtype list
         """
+        print('Getting API Resources')
```

Comment:
This log would be more useful if it said what resources it was trying to access (i.e the path argument)